src/main/resources/emails/approvalNeeded.ftlh

```diff
@@ -126,6 +126,13 @@ Dear ${approvalStepName},
   <strong>Engagement date and location:</strong> ${(engagementDateFormatter.format(report.engagementDate))!} @ ${(report.loadLocation(context).get().name)!}
 </div>
 
+<#if engagementsIncludeTimeAndDuration>
+  <div>
+    <strong>Duration (minutes): </strong> ${(report.duration)!}
```

Comment:
Remove the space before the close tag: `<strong>Duration (minutes):</strong> ${(report.duration)!}`